addons/product/models/product_template.py

```diff
@@ -54,6 +54,16 @@ def _read_group_categ_id(self, categories, domain):
         help='A storable product is a product for which you manage stock. The Inventory app has to be installed.\n'
              'A consumable product is a product for which stock is not managed.\n'
              'A service is a non-material product you provide.')
+    service_tracking = fields.Selection(selection=[
```

Comment:
in fact, this change cannot be done after freeze as it requires an upgrade script :/

Reply:
We can add/update upgrade script in 17.3 folder in master upgrade branch :thinking: